### ISSUE TITLE
[Bug]: (Program Crash)

### ISSUE THREAD

**Issue by marginal23326** [ThioJoe/YT-Spammer-Purge#341]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

Well, I was trying to run the program, but then it immediately crashed.

### Release version

2.7.3

### Steps to reproduce

1. Run the program by double clicking it.

### What platform are you seeing this problem on?

Windows

### Relevant log output

```shell
Traceback (most recent call last):
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 3599, in <module>
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 2734, in main
KeyError: 'release_channel'
[31776] Failed to execute script 'YouTubeSpammerPurge' due to unhandled exception!

[process exited with code 1]
```

### Screenshots

```shell
![SpammerPurgeError](https://user-images.githubusercontent.com/58261815/148438552-21533ea4-1659-47c8-8468-e1c6f113bb47.png)

```

**Comment by Firecul** [comment]
I think you have an old verison of the .ini, the same thing has happened to me. Unless ThioJoe says otherwise, it's easiest to try deleting (or renaming) your config file then trying to run the program again. If it runs,regenerate the config (option 5) and change your settings again.

**Comment by ThioJoe** [comment]
Yea it's because of an old config version, just delete the file and run it again

**Comment by ThioJoe** [comment]
Ok should be fixed with release 2.7.4
